R/SupervisedTask.R

```diff
@@ -1,13 +1,13 @@
-makeSupervisedTask = function(type, data, target, weights, blocking, fixup.data, check.data, spatial) {
-  task = makeTask(type = type, data = data, weights = weights, blocking = blocking, fixup.data = fixup.data, check.data = check.data, spatial = spatial)
+makeSupervisedTask = function(type, data, target, weights, blocking, fixup.data, check.data, coordinates) {
+  task = makeTask(type = type, data = data, weights = weights, blocking = blocking, fixup.data = fixup.data, check.data = check.data, coordinates = coordinates)
```

Comment:
Should the `coordinates` argument be checked here? Presumably it needs to have a certain structure.

Reply:
This is done [here](https://github.com/mlr-org/mlr/blob/aeda74198da8b51421db795e4666e591dd884b5c/R/Task.R#L148-L154). I think its sufficient?

Reply:
But the columns need to have specific names, don't they?

Reply:
No, not anymore.